packages/matter-node.js/src/Device.ts

```diff
@@ -43,22 +43,29 @@ import { AdminCommissioningHandler } from "./matter/cluster/server/AdminCommissi
 import { NetworkCommissioningHandler } from "./matter/cluster/server/NetworkCommissioningServer";
 import { AttestationCertificateManager } from "./matter/certificate/AttestationCertificateManager";
 import { CertificationDeclarationManager } from "./matter/certificate/CertificationDeclarationManager";
+import { StorageNode } from "./persistence/StorageNode";
+import { PersistenceManager } from "./persistence/PersistenceManager";
 
 const logger = Logger.get("Device");
 
+const storage = new StorageNode(getParameter("file") || "device.json")
+
 class Device {
     async start() {
         logger.info(`node-matter`);
 
+        const persistenceManager = new PersistenceManager(storage);
+        await persistenceManager.initialize();
+
         const deviceName = "Matter test device";
         const deviceType = 257 /* Dimmable bulb */;
-        const vendorName = "node-matter";
-        const passcode = 20202021;
-        const discriminator = 3840;
+        const vendorName = "matter-node.js";
+        const passcode = getIntParameter("passcode") ?? 20202021;
```

Comment:
Shouldn't the sequence be:
- if passed via command line, use it
- else try to load from storage
- else use default

Reply:
Yes, it is not yet stored in persistence ... I can add that

Reply:
added